src/com/pentalog/pentastagiu/homework2/ConsoleApp/MainApp.java

```diff
@@ -0,0 +1,18 @@
+package com.pentalog.pentastagiu.homework2.ConsoleApp;
+
+public class MainApp {
+    public static void main(String[] args) {
+        Customer FirstCustomer=new Customer("Tomoiaga Horatiu","SILVER");
+        Candy TootsieRoll=new Candy(20,"TootsieRoll",10);
+        Product Masa=new Product(10,"MASA");
+        Book Wu=new Book(30,"Wu","Mark");
+        Basket FirstBasket=new Basket();
+        FirstBasket.customer=FirstCustomer;
+        FirstBasket.AddProduct(TootsieRoll);
+        FirstBasket.AddProduct(Masa);
+        System.out.println("You have to pay "+ FirstBasket.getTotalPrice());
+        FirstCustomer.setMembership("GOLD");
```

Comment:
cand ai o colectie limitata de valori, se recomanda folosirea unui enum, e mai putin error prone(cineva poate scrie din greseala "gold" in loc de varianta uppercase si va genera alta logica decat te astepti)